crates/resolver/src/async_resolver/background.rs

```diff
@@ -107,7 +108,7 @@ impl Task {
             } else {
                 return LookupIpFuture::ok(
                     self.client_cache.clone(),
-                    Lookup::new_with_max_ttl(Arc::new(vec![ip_addr])),
+                    Lookup::new_with_max_ttl(Query::new(), Arc::new(vec![ip_addr])),
```

Comment:
I'm not sure what the appropriate query to use in these cases are, so I just used the empty/default query.